modules/calib3d/src/chessboard.cpp

```diff
@@ -207,15 +202,16 @@ bool isPointOnLine(cv::Point2f l1,cv::Point2f l2,cv::Point2f pt,float min_angle)
 }
 
 // returns how many tests fails out of 10
-int testPointSymmetry(cv::Mat mat,cv::Point2f pt,float dist,float max_error)
+int testPointSymmetry(const cv::Mat& mat,cv::Point2f pt,float dist,float max_error)
 {
     cv::Rect image_rect(int(0.5*dist),int(0.5*dist),int(mat.cols-0.5*dist),int(mat.rows-0.5*dist));
     cv::Size size(int(0.5*dist),int(0.5*dist));
     int count = 0;
     cv::Mat patch1,patch2;
     cv::Point2f center1,center2;
-    for(double angle=0;angle <= CV_PI;angle+=CV_PI*0.1)
+    for (int angle_i = 0; angle_i < 10; angle_i++)
     {
+        double angle = angle_i * (CV_PI * 0.1);
```

Comment:
@D-Alex Is this loop still correct? (`<=` condition has been replaced to `<` to get exact 10 "tests")

Reply:
Looks ok. PI was already tested during the first iteration and should not be tested twice like in the previous implementation.